phalcon/diinterface.zep

```diff
@@ -64,7 +64,7 @@ interface DiInterface extends \ArrayAccess
 	 * @param boolean shared
 	 * @return \Phalcon\Di\ServiceInterface
 	 */
-	public function attempt(string! name, definition, boolean shared = false) -> <ServiceInterface>;
+	public function attempt(string! name, definition, boolean shared = false) -> <ServiceInterface> | boolean;
```

Comment:
Why ?

Reply:
https://github.com/phalcon/cphalcon/blob/master/phalcon/di.zep#L158

Reply:
Consider refactor concrete class to always return ServiceInterface